### DISCUSSION TITLE
Managers Page Not Loading and Corrupting Rest of Site

### DISCUSSION THREAD

**Discussion by dawghouse73** [nmelhado/league-page#217] [Q&A] [answered]
I've updated my new league info and added details Managers section of the league_info.js. 
I've tried with one a manager section and all 12, but same results. 

When I go to the Managers page in the league site, it loads nothing. It then causes the rest of the site to fail to load and I'm forced to refresh my browser to get anything to work - except the Managers page. 

So, issues:
1. Managers page doesn't load with any information
2. Clicking the Managers page causes the rest of the site to stop loading (even if previously loaded in that session)
3. No pages load after I click Managers, except those links taking me to external sites (Sleeper, etc.)

All menu shortcuts work - except Managers. Both top nav and bottom nav. 

It's weird. I would assume if there was an issue with the information in the league_info.js, it wouldn't deploy in the first place. 

UPDATE: I've also noticed that if I copy/paste the league URL/managers into a new browser, I get the Hut, Hut Error page. I can continue to navigate around the rest of the site. But as soon as I try to access that Managers page from the menus, it causes the site to stop loading entirely. 
_NOTE: I'm not sure if it's 100% directly related, but I am also getting a Failed Production Deployment email every so often, even though I'm not deploying anything. And no changes have been since the last successful deployment._ 

Here's what I see when I click on the Managers shortcut in either the top nav or bottom nav:
![image](https://github.com/nmelhado/league-page/assets/133917715/21d7031a-2f74-4108-8969-3d7bfef345f4)

**Comment by nmelhado** [comment, marked as answer]
Make sure you're including all fields that aren't listed as optional, including rival. If it's not optional, then the code will assume it exists.

**Reply by dawghouse73** [reply]
Thanks - forgot to mark this off previously. 
Top notch as usual!
